scripts/generateRequires.mjs

```diff
@@ -0,0 +1,41 @@
+import { readdir, writeFile } from 'node:fs/promises';
+import { Constants } from '../src/index.js';
+
+async function writeWebsocketHandlerImports() {
+  const lines = ["'use strict';\n", 'const handlers = Object.fromEntries(['];
+
+  for (const name of Object.keys(Constants.WSEvents)) {
+    lines.push(`  ['${name}', require('./${name}')],`);
+  }
+
+  lines.push(']);\n\nmodule.exports = handlers;\n');
+
+  const outputFile = new URL('../src/client/websocket/handlers/index.js', import.meta.url);
+
+  await writeFile(outputFile, lines.join('\n'));
+}
+
+async function writeClientActionImports() {
+  const lines = ["'use strict';\n", 'class ActionsManager {', '  constructor(client) {', '    this.client = client;\n'];
+
+  const actionsDirectory = new URL('../src/client/actions', import.meta.url);
+  for (const file of await readdir(actionsDirectory)) {
```

Comment:
I don't think `readdir` gives any guarantees about the order, I suggest sticking a `.sort()` in there to make sure that everyone gets the same result

Reply:
You're right, addressed in d6c13168.